src/components/greeting.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const Greeting = () => {
+  const message = useSelector((state) => state.greetings.greating);
```

Comment:
- [ ] You did a great job working on this milestone, however, you have a typo that makes this `message` variable undefined: you spelled `greating` instead of `greeting`. Please fix this! 🤓 🔧